web_src/css/repo.css

```diff
@@ -531,6 +531,9 @@ td .commit-summary {
   line-height: 32px;
   vertical-align: middle;
   color: var(--color-text-light);
+  overflow-wrap: break-word;
+  max-width: 100%;
+  padding-right: 1rem;
```

Comment:
Why it needs these two new styles?

I don't see difference.

Reply:
Without it, a unbroken line will wrap exactly at the end of the element, which looks visually unpleasing without any spacing.

Reply:
* `padding-right: 1rem;`: add extra space for wrapped content?
* `max-width: 100%;`: what does this do?